cdap-standalone/src/main/resources/cdap-site.xml

```diff
@@ -68,6 +68,14 @@
     </description>
   </property>
 
+  <property>
```

Comment:
This should goes into cdap-default.xml?

Reply:
If you use relative path, shouldn't it works in both?

Reply:
Hmm I looked into this, but the same relative path wouldn't work in both cases. I just followed examples from other properties like `bootstrap.file` or `app.artifact.dir`, where the path is relative in cdap-site and absolute in cdap-default. We also want to have the property here so that user can easily overwrite the value, instead of having to manually add the entire property themselves.